src/ApiClient/AnnotationsClient.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace eLife\ApiSdk\ApiClient;
```

Comment:
The change in namespace makes this backward incompatible, at least for the journal case where many classes are visible and mentioned directly:
https://github.com/search?q=repo%3Aelifesciences%2Fjournal+%22eLife%5CApiClient%22&type=code

Reply:
Yes that's true, though it is worth saying it is only the `eLife\ApiClient\ApiClient` namespace that has moved, so there is only one file affected in Journal.